src/api/requests/fetchFilingSubmissionLatest.ts

```diff
@@ -91,11 +91,13 @@ async function retryRequestWithDelay(
 
 /** Used in `useGetSubmissionLatest` to long poll for validation after an upload * */
 function shouldRetry(response: AxiosResponse<SubmissionResponse>): boolean {
-  // Check if the response has a 'state' property equal to "VALIDATION_IN_PROGRESS"
+  // Check if the response has a 'state' property equal to "VALIDATION_IN_PROGRESS" or "SUBMISSION_UPLOADED"
   // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition, @typescript-eslint/no-unused-expressions, prettier/prettier
-  return (response.data?.state && response.data.state) === fileSubmissionState.VALIDATION_IN_PROGRESS;
+  return (response.data?.state && response.data.state) === FileSubmissionState.VALIDATION_IN_PROGRESS || (response.data?.state && response.data.state) === FileSubmissionState.SUBMISSION_UPLOADED;
```

Comment:
Could this be simplified?

```suggestion
  return [FileSubmissionState.VALIDATION_IN_PROGRESS, FileSubmissionState.SUBMISSION_UPLOADED].includes(response.data?.state);
```